mlir/utils/performance/perfRunner.py

```diff
@@ -575,6 +574,7 @@ def fromCommandLine(cls, argv, arch):
         n = None
         transA = None
         transB = None
+
```

Comment:
Could you remove space changes? (Unless it's important that the space stays there)